my-app/src/components/menu.js

```diff
@@ -63,14 +63,14 @@ export default class MenuSecondary extends Component {
           active={activeItem === 'Diagramy'}
           onClick={this.handleItemClick}
         />
-        
+         
         <Menu.Menu position='right'>
-          
-      <div className="language" style ={{width:'100%', padding:'2rem, 0'}}>
-      <button onClick={()=>handleClick('pl')}><Pl/></button>
-      <button onClick={()=>handleClick('en')}><En/></button>
-      </div>     
-      <Logout></Logout> </Menu.Menu>
+
+          <div className="language" style={{ width: '100%', padding: '2rem, 0' }}>
+            <button onClick={() => handleClick('pl')}><Pl /></button>
```

Comment:
please move those handlers to separate methods

Reply:
done